### PR TITLE
taskwarrior-sync: add service module

### PR DESCRIPTION
**Pull request by pacien**
This PR adds a service module for Taskwarrior periodic sync.

CC: @minijackson

### CODE REVIEW

File: modules/modules.nix [hunk 1]
```diff
@@ -124,6 +124,7 @@ let
     (loadModule ./services/syncthing.nix { })
     (loadModule ./services/taffybar.nix { })
     (loadModule ./services/tahoe-lafs.nix { })
+    (loadModule ./services/taskwarrior-sync.nix { condition = hostPlatform.isLinux; })
```
Comment (pacien) [inline modules/modules.nix]: I'm not sure whether this module should be named `taskwarrior` instead for consistency with `programs/taskwarrior`. `taskwarrior-sync` seems more explicit to me though.

File: modules/services/taskwarrior-sync.nix [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          <refentrytitle>systemd.time</refentrytitle>
+          <manvolnum>7</manvolnum>
+        </citerefentry>
+        for more information about the format.
+      '';
+    };
+  };
+
+  config = let
+    cfg = config.services.taskwarrior-sync;
```
Comment (rycee) [inline modules/services/taskwarrior-sync.nix]: Please move this to the top to match the other modules.

**Comment by rycee** [comment]
Looks good to me, thanks for the contribution. If the comment is resolved then I can merge.

File: modules/services/taskwarrior-sync.nix (hunk 2 above)
Comment (pacien) [reply modules/services/taskwarrior-sync.nix]: Done and amended.

**Comment by rycee** [comment]
Excellent! Rebased to master in cc0cd538e62ce5e220ae296e65342ba258d186f8.

**Pull request closed without merging**